Add jest

pyromaniackeca opened the pull request:
- Add jest as dev dependency
- Add dummy test to make sure jest is working

Perhaps we also want to add:
- `babel-jest`
- `react-test-renderer`

Not sure if we need them, but we can also add them later as necessary.

I also found a recommendation for a better initial setup as described [here](https://www.reactnative.guide/7-testing/7.1-jest-setup.html).
But again, we might not need it and I'd keep it as simple as possible if we don't.

Let me know what you think.

pyromaniackeca committed:
Add jest

pyromaniackeca committed:
Add dummy test

AnaPavlov commented on package.json:
```diff
@@ -18,7 +19,11 @@
+    "jest": "^27.4.7"
+  },
+  "jest": {
+    "preset": "react-native"
```
Since we are using Expo framework, I would suggest to set "preset" value to "jest-expo", just to keep things consistent. 

Check out here https://docs.expo.dev/guides/testing-with-jest/

AnaPavlov commented:
I agree, let's keep it simple and we'll add more things to the config only if it's needed.

pyromaniackeca committed:
Add jest-expo

pyromaniackeca commented:
@AnaPavlov I've updated the PR. I've also added [this](https://docs.expo.dev/guides/testing-with-jest/#jest-configuration) to the config. Not 100% sure what it does, but it seems like something we'll need.
